src/posix.mak

```diff
@@ -90,11 +90,12 @@ else
   # Auto-bootstrapping, will download dmd automatically
   # Keep var below in sync with other occurrences of that variable, e.g. in circleci.sh
   HOST_DMD_VER=2.072.2
+  HOST_DMD_YEAR=2016
   HOST_DMD_ROOT=$(TMP)/.host_dmd-$(HOST_DMD_VER)
   # dmd.2.072.2.osx.zip or dmd.2.072.2.linux.tar.xz
   HOST_DMD_BASENAME=dmd.$(HOST_DMD_VER).$(OS)$(if $(filter $(OS),freebsd),-$(MODEL),)
   # http://downloads.dlang.org/releases/2.x/2.072.2/dmd.2.072.2.linux.tar.xz
-  HOST_DMD_URL=http://downloads.dlang.org/releases/2.x/$(HOST_DMD_VER)/$(HOST_DMD_BASENAME)
+  HOST_DMD_URL=https://s3-us-west-2.amazonaws.com/downloads.dlang.org/releases/$(HOST_DMD_YEAR)/$(HOST_DMD_BASENAME)
```

Comment:
> How long is that link going to keep working? It looks awfully fragile, with apparently having the datacenter location in it.

For as long as the S3 bucket is alive (S3 buckets are bound to a datacenter and can't be moved to a different location or renamed).

> The solution here is rather to either enable proper SSL for downloads.dlang.org,

See also: https://trello.com/c/mWHZxwo5/89-https-for-s3-downloadsdlangorg

>  or just add a SHA256 check to the downloads.

This is going to be annoying for updates and ensuring that `sha256sum` is installed on all machines. Also we can't do the streaming extraction anymore.